internal/api/registry/manifests_test.go

```diff
@@ -573,6 +574,31 @@ func TestManifestRequiredLabels(t *testing.T) {
 			ExpectStatus: http.StatusCreated,
 			ExpectHeader: test.VersionHeader,
 		}.Check(t, h)
+
+		//check that the labels_json field is populated correctly in the db
+		var dbManifests []keppel.Manifest
+		_, err = db.Select(&dbManifests,
+			`SELECT * FROM manifests`,
+		)
+		if err != nil {
+			t.Fatal(err.Error())
+		}
+		if len(dbManifests) != 1 {
+			t.Fatalf("expected 1 manifest entry in db, got: %d", len(dbManifests))
+		}
+
+		var actual map[string]string
+		err = json.Unmarshal([]byte(dbManifests[0].LabelsJSON), &actual)
+		if err != nil {
+			t.Fatal(err.Error())
+		}
+		expected := map[string]string{
+			"bar": "is there",
+			"foo": "is there",
+		}
+		if !assert.DeepEqual(t, "labels_json", actual, expected) {
+			t.Fatal("expected labels_json to be correctly populated during image push")
+		}
```

Comment:
`assert.DeepEqual()` already prints an error message if the assertion fails, so the additional custom message is not required.
```suggestion
		assert.DeepEqual(t, "labels_json", actual, expected)
```